leader-election-impl/src/main/java/com/palantir/paxos/PaxosLatestRoundVerifierImpl.java

```diff
@@ -63,7 +64,7 @@ private boolean acceptorAgreesIsLatestRound(PaxosAcceptor acceptor, long round)
         try {
             return round >= acceptor.getLatestSequencePreparedOrAccepted();
         } catch (Exception e) {
-            if (!onlyLogOnQuorumFailure.get()) {
+            if (isPicked()) {
```

Comment:
I'd recommend a more descriptive name for the method, such as `shouldLogFailure`, but especially as this is temporary code, it's not a big deal.